build.sbt

```diff
@@ -2118,6 +2118,7 @@ lazy val `engine-common` = project
   .enablePlugins(JPMSPlugin)
   .settings(
     frgaalJavaCompilerSetting,
+    scalaModuleDependencySetting,
```

Comment:
The comment for this module says:

> engine/common project contains classes that are necessary to configure GraalVM's polyglot context. Most specifically it contains `ContextFactory`. As such it needs to depend on `org.graalvm.polyglot` package. Otherwise its dependencies shall be limited - no JSON & co. please. 

This change adds a dependency on Scala. Knowing we want to get rid of Scala this probably contra-productive (effectively all our `runtime*` projects will have transitive Scala dependency regardless whether they need it or not). Isn't there some other module that already depends on Scala where to put `ScalaConversion` utility?

Reply:
Famous words: let's fix it later.

Reply:
EDIT: It seems that it is possible to move `ScalaConversions` to `scala-libs-wrapper`. I will try that.

Reply:
Moved to `scala-libs-wrapper` in 417843dc850d49075f89970f4284749e15118e5d